nl2fetch.py

```diff
@@ -47,6 +49,7 @@ def nl2_get_telemetry(self, packet: Packet) -> None:
             # if this is the first frame being received,
             # set the last position to the current position
             # for purposes of calculating velocity
+            self.play_mode = True
             self.last_pos = self.last_pos or (data.position_x, data.position_y, data.position_z)
```

Comment:
Nit: Can you move self.play_mode under self.last_pos? I believe the comment above was supposed to be for self.last_pos so it would make sense to have the variable declaration right underneath it